.travis.yml

```diff
@@ -4,30 +4,24 @@ os:
   - linux
   - osx
```

Comment:
Test on Windows too now
```suggestion
  - osx
  - windows
```

Reply:
Do we care about testing this on windows?